src/Symfony/Component/VarDumper/Caster/DOMCaster.php

```diff
@@ -11,6 +11,20 @@
 
 namespace Symfony\Component\VarDumper\Caster;
 
+use Dom\Attr;
+use Dom\CharacterData;
+use Dom\DocumentType;
+use Dom\Element;
+use Dom\Entity;
+use Dom\Exception;
+use Dom\HTMLDocument;
+use Dom\Implementation;
+use Dom\Node;
+use Dom\Notation;
+use Dom\ProcessingInstruction;
+use Dom\Text;
+use Dom\XMLDocument;
+use Dom\XPath;
```

Comment:
let's keep using no `use` don't you think? I feel like it's more readable

Reply:
done